python/cudf_polars/cudf_polars/typing/__init__.py

```diff
@@ -108,7 +108,7 @@ def view_current_node(self) -> PolarsIR:
         """Convert current plan node to python rep."""
         ...
 
-    def get_schema(self) -> Schema:
+    def get_schema(self) -> PolarsSchema:
```

Comment:
I think this was incorrect earlier, though it ended up not mattering since it was only used in one place (`Translator.translate_ir`) with the correct runtime types.